Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -345,7 +319,7 @@ def get_metainfo(self, infohash_or_magnet, callback, timeout=30, timeout_callbac
             elif infohash not in self.metainfo_requests:
                 # Flags = 4 (upload mode), should prevent libtorrent from creating files
                 atp = {'save_path': self.metadata_tmpdir, 'duplicate_is_error': True, 'paused': False,
-                       'auto_managed': False, 'flags': 4}
+                       'auto_managed': False, 'upload_mode': True}
```

Comment:
Maybe keep both methods? Or is flags incompatible with newer versions of libtorrent?